site/en/tutorials/load_data/images.ipynb

```diff
@@ -927,7 +928,7 @@
         "id": "CboYya6LmdQI"
       },
       "source": [
-        "So before the passing it to the MobilNet model, we need to convert the input from a range of `[0,1]` to `[-1,1]`."
+        "So before passing data to the MobileNet model, we need to convert the input from a range of `[0,1]` to `[-1,1]`."
```

Comment:
Not your doing but since you're in here, can you replace "we" with "you" ?
This is the Google developer docs style: https://developers.google.com/style/person

Reply:
Thank you for the link, I've updated the notebook accordingly!